code/src/App.js

```diff
@@ -1,9 +1,84 @@
-import React from 'react'
+//Outer dependencies
+import React, { useState, useEffect} from 'react'
+
+//Local dependencies 
+import  MessageForm  from "./components/MessageForm"
+import MessageList from "./components/MessageList"
+
+import { API_URL, API_URL_LIKES } from './reusable/url'
+
 
 export const App = () => {
+  const [messageList, setMessageList] = useState([])
+  const [newMessage, setNewMessage] = useState('')
+
+  useEffect(() => {
+    fetchMessageList()
+  }, []) // If it should always be called we do not set a second argument with a comma , 
+
+  const fetchMessageList = () => {
+    fetch(API_URL)
+      .then(res => res.json())
+      .then(messages => setMessageList(messages))
+      .catch(err => console.error(err));
+  }
+
+  const handleNewMessageChange = (event) => {
+    setNewMessage(event.target.value);
+  }
+
+  const handleFormSubmit = (event) => {
+    event.preventDefault(); 
+
+    setNewMessage('') //empty textarea after submit
+
+    fetch(API_URL, {
+      method: 'POST', 
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({ message: newMessage }) 
+    })
+    .then((res) => res.json ()) //what happens in this section? how is it making real time after submit
```

Comment:
//what happens in this section? how is it making real-time after submitting

the response is in `json` format. `res.json()` parse it to the normal js code.